services/httpoverrpc/client/client.go

```diff
@@ -136,16 +136,18 @@ func (p *proxyCmd) Execute(ctx context.Context, f *flag.FlagSet, args ...interfa
 			sendError(httpResp, http.StatusBadRequest, err)
 			return
 		}
-		req := &pb.LocalhostHTTPRequest{
+		req := &pb.HostHTTPRequest{
 			Request: &pb.HTTPRequest{
 				RequestUri: httpReq.RequestURI,
 				Method:     httpReq.Method,
 				Headers:    reqHeaders,
 				Body:       body,
 			},
-			Port: int32(port),
+			Port:     int32(port),
```

Comment:
let's add validation for port number

Reply:
Yes, I will verify the port number fall into the range of [0~65535]